README.md

```diff
@@ -99,6 +99,29 @@ filter to the job:
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
 ```
 
+### Generating and uploading attestations
```

Comment:
Can we mention somewhere that the input may change without prior notice and is not to be considered stable API at this point? I want to avoid having to go through a deprecation cycle for such feature previews.

Reply:
Tweaked the existing `IMPORTANT` callout to emphasize this!